packages/contracts/src/dollar/DirectGovernanceFarmer.sol

```diff
@@ -136,60 +260,96 @@ contract DirectGovernanceFarmer is ReentrancyGuard {
      * @param stakingShareId Staking Share Id to withdraw
      * @param token Token to withdraw to : DAI, USDC, USDT, 3CRV or Ubiquity Dollar
      */
-    function withdraw(
-        uint256 stakingShareId,
-        address token
-    ) external nonReentrant returns (uint256 tokenAmount) {
-        
+    function withdraw(uint256 stakingShareId, address token)
```

Comment:
Limiting user to withdraw only the coin is quite limiting and forces them to incur some slippage

Reply:
also not really part of that issue but it makes sense to let people withdraw all tokens (why wasn't part of the first version btw ?). I will add it nonetheless